oxidation/crates/parsec_api_types/src/lib.rs

```diff
@@ -11,6 +11,7 @@ mod id;
 mod invite;
 mod manifest;
 mod message;
+mod protocol;
```

Comment:
`protocol` is a name we already use for `parsec.api.protocol` (i.e. data format for communication between client and server), so I'd rather use another one